### PR TITLE
Add CZNC::IsBindHostAllowed()

### PR DESCRIPTION
**Pull request by jpnurmi**

### CODE REVIEW

File: src/znc.cpp [hunk 1]
```diff
@@ -338,6 +338,18 @@ bool CZNC::IsHostAllowed(const CString& sHostMask) const {
 	return false;
 }
 
+bool CZNC::IsBindHostAllowed(const CString& sBindHost) const {
```
Comment (DarthGandalf) [inline src/znc.cpp]: This is as useless as it was before this PR.
If sBindHost is empty, OS is free to choose *any* IP and connect from it. And user is always free to make bindhost empty regardless of this list.

**Comment by DarthGandalf** [comment]
Also see #983

**Comment by jpnurmi** [comment]
So what do you suggest instead? My intention is not to fix any bindhost misbehavior, or change any behavior at all, but just share that piece of code that is already repeated in five different places within the ZNC codebase. Then I don't have to repeat it once again in https://github.com/jpnurmi/znc-settings...

**Comment by DarthGandalf** [comment]
Just remove this feature? Right now it does more harm than good, and adding more API for it won't help.
If you have some idea how to make such list of hosts to be actually useful, the API can follow (maybe it would even stay the same)

**Pull request closed without merging**